src/frontend/src/components/common/Checkbox/index.tsx

```diff
@@ -47,24 +45,7 @@ const Checkbox = ({
         )}
       </label>
       {!tooltip ? null : (
-        <>
-          <FontAwesomeIcon
-            data-tip={tooltipText}
-            data-for={value}
-            className="cb-info-icon"
-            icon={faInfoCircle}
-          />
-          <Tooltip
-            className="tooltip"
-            id={value}
-            place="right"
-            effect="solid"
-            arrowColor="#1A5A96"
-            getContent={() => {
-              return <div>{tooltipText}</div>;
-            }}
-          />
-        </>
+        <Tooltip id={value} content={tooltipText} direction="right" />
```

Comment:
Won't having id as value mean it will change every time value of the checkbox is changed? It should maybe be label

Reply:
> Won't having id as value mean it will change every time value of the checkbox is changed? It should maybe be label

yes right by changing it to 'label,' we could prevent the 'value' of the checkbox from changing. I have changed it!